code-challenges.test.js

```diff
@@ -28,16 +32,43 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+const num1 = 15 
 // Expected output: "15 is divisible by three"
 const num2 = 0
 // Expected output: "0 is divisible by three"
 const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
 
+describe("dividend", () => {
+    test("takes a number as an argument and decides if the number is evenly divisble by three or not")
+    const num1 = 15
+    const num2 = 0
+    const num3 = -7
+    expect(dividend(num1)).toEqual("num1 is divisible by 3")
+    expect(dividend(num2)).toEqual("num2 is divisible by 3")
+    expect(dividend(num3)).toEqual("num3 is not divisible by 3")
+})
```

Comment:
There may be some syntax that needs adjusting, but the overall structure looks pretty good!